server/database/modules/campaign.ts

```diff
@@ -0,0 +1,36 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../config/connection';
+import { category, donation, capon } from './index';
+
+const campaign = sequelize.define('campaigns', {
```

Comment:
best `sequelize` naming convention that model name should be `Upper Case`
![image](https://user-images.githubusercontent.com/71943250/164517836-92494d71-98ad-40ec-aa40-91748ccb3db1.png)